docker/Dockerfile.public_api

```diff
@@ -10,11 +10,16 @@ COPY reactome_analysis_api-*.whl /usr/apps/
 # prepare data structure for the search index
 RUN mkdir -p /data/search_index
 ENV SEARCH_INDEX_PATH /data/search_index
+ENV SEARCH_INDEX_WHITELIST /data/search_index/whitelist.txt 
+ENV SEARCH_INDEX_BLACKLIST /data/search_index/blacklist.txt
 
 # install the apps
 RUN pip3 install /usr/apps/reactome_analysis_utils*.whl && \
     pip3 install /usr/apps/reactome_analysis_api-*.whl
 
+# update whitelist for valid GREIN datasets
+RUN update_whitelist
```

Comment:
I don't quite understand where the command `update_whitelist` comes from

I guess you wanted to run `update_grein.py`?

Anything after `RUN` is actually executed in a shell within the docker container. Therefore, you first would need to add the python script to the conteiner. F.e.

```
# add the script to update the whitelinst
COPY update_grein.py /tmp/
# update the whitelist
RUN python3 /tmp/update_grein.py
```

**BUT**

The main issue with this approach is, that the whitelist within the GitHub repository is not changed. And once new datasets are added, this will accumulate. Therefore, a probably better solution is, to add the call to the update script to our `build_images.sh` script. This then ensures, that the whitelist (within our working directory) is updated each time we create a new version of the image.

What do you think?